specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyTrackedResources.json

```diff
@@ -368,6 +368,10 @@
       "enum": [
         "default"
       ],
+      "x-ms-enum": {
+        "name": "PolicyTrackedResourcesResourceType",
+        "modelAsString": false
```

Comment:
why is this false?

Reply:
All of these enums have once constant value, so I guess it's fine(?)